hidabc/hidapi.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: MIT
+
+import functools
+
+from typing import List
+
+import hid
+
+import hidabc
+
+
+class HidapiInterface(hidabc.ExtendedInterface):
+    def __init__(self, path: str) -> None:
```

Comment:
Please make this keyword only, as in the future we may want to add support for opening via the vid and pid.

```suggestion
    def __init__(self, *, path: str) -> None:
```